compose-builder/add-security.yml

```diff
@@ -257,12 +214,40 @@ services:
       ROUTES_DEVICE_VIRTUAL_HOST: device-virtual
     volumes:
       - edgex-init:/edgex-init:ro,z
+      - vault-config:/vault/config:z
+      - nginx-templates:/etc/nginx/templates
+      - nginx-tls:/etc/ssl/nginx
       - /tmp/edgex/secrets/security-proxy-setup:/tmp/edgex/secrets/security-proxy-setup:ro,z
       - consul-acl-token:/tmp/edgex/secrets/consul-acl-token:ro,z
     depends_on:
       - security-bootstrapper
       - secretstore-setup
-      - kong
+    security_opt: 
+      - no-new-privileges:true
+
+  authproxy:
```

Comment:
how about `proxyauth` or `proxy-auth` which is align with your microservice(binary) name ?

Reply:
Changed to proxy-auth.